WordPress/src/main/java/org/wordpress/android/ui/reader/actions/ReaderBlogActions.java

```diff
@@ -442,19 +442,22 @@ public void onErrorResponse(VolleyError volleyError) {
         WordPress.sRequestQueue.add(request);
     }
 
-    /*
-     * block a blog - result includes the list of posts that were deleted by the block so they
-     * can be restored if the user undoes the block
-     */
-    public static BlockedBlogResult blockBlogFromReader(final long blogId, final ActionListener actionListener) {
```

Comment:
I've split the `blockBlogFromReader` method into two methods `blockBlogFromReaderLocal` and `blockBlogFromReaderRemote `.